indico/modules/auth/util.py

```diff
@@ -51,6 +56,41 @@ def load_identity_info():
     return info
 
 
+def register_user(email, extra_emails, user_data, identity_data, settings, from_moderation=False):
```

Comment:
Specific from the `users` module perspective. Still just an operation from the `auth` module's.